package.json

```diff
@@ -11,7 +11,7 @@
     "test": "jest",
     "lint": "eslint --ext .js,.ts,.tsx .",
     "format": "prettier --write **/*.{js,ts,tsx,json,md}",
-    "compile": "tsc --noEmit",
+    "compile": "tsc --noEmit -t es2015",
```

Comment:
Why do you use explicit target? There should be target declaration in tsconfig.json

Reply:
Decided to use es2015 in root tsconfig and set es5 target for reselect-utils build script.